docs/reference/commandline/service_update.md

```diff
@@ -74,6 +74,7 @@ Options:
       --rollback-max-failure-ratio float   Failure rate to tolerate during a rollback
       --rollback-monitor duration          Duration after each task rollback to monitor for failure
                                            (ns|us|ms|s|m|h) (default 0s)
+      --rollback-order string              Rollback order ("start-first"|"stop-first") (default "stop-first")
```

Comment:
Stating default in `service update` command is a little misleading. If you don't specify the flag, it keeps existing value.